docs/contributing/contributing-code/contributing-code-prerequisites/README.md

```diff
@@ -1,16 +1,49 @@
 # Repository Prerequisites
 
-This page lists the prerequisites for working with the repository. Most contributors should start with the basic prerequisites. Depending on the task you need to perform, you may need to install more tools.
+This page lists the prerequisites for working with the repository. In general, you have three options to get your development setup up and running:
+
+- Remotely using GitHub Codespaces
+- Locally using Visual Studio Code and dev containers leveraging
+- Locally by installing all prerequisites on your
+
+We will walk you through the options in the following sections.
+
+> 📝 **Tip** - We recommend the usage of GitHub Codespaces of local dev containers as they are the most convenient way to get started.
+
+Depending on the task you need to perform, you may need to install more tools, but basic prerequisites should be sufficient for most contributors to get started.
 
 ## Operating system
 
 We support developing on macOS, Linux and Windows with [WSL](https://docs.microsoft.com/windows/wsl/install).
 
 ## Asking for help
 
-If you get stuck installing any of our dependencies, please ask for help in our [forum](https://discordapp.com/channels/1113519723347456110/1115302284356767814).
+If you get stuck with any development setup option, please ask for help in our [forum](https://discordapp.com/channels/1113519723347456110/1115302284356767814).
+
+## Editors
+
+If you don't have a code editor set up for Go, we recommend VS Code. The experience with VS Code is high-quality and approachable for newcomers.
+
+Alternatively, you can choose whichever editor you are most comfortable for working on Go code. Feel free to skip this section if you want to make another choice.
 
-## Basic Prerequisites
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Go extension](https://marketplace.visualstudio.com/items?itemName=golang.go)
+
+Install both of these and then follow the steps in the *Quick Start* for the Go extension.
+
```

Comment:
Can you put a small section here linking to the devcontainer setup? https://code.visualstudio.com/docs/devcontainers/tutorial

Reply:
Done